account_invoice_pricelist/view/account_invoice_view.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    © 2015-2016 GRAP <http://www.grap.coop>
+    © 2017 Therp BV <http://therp.nl>.
+    License AGPL-3.0 or later <http://www.gnu.org/licenses/agpl.html>.
+-->
+<openerp>
+    <data>
+
+        <!-- Model: account.invoice -->
+        <record id="view_account_invoice_customer_form" model="ir.ui.view">
+            <field name="model">account.invoice</field>
+            <field name="inherit_id" ref="account.invoice_form"/>
+            <field name="arch" type="xml">
+                <field name="state" position="before">
+                    <button
+                        type="object"
+                        name="button_update_prices_from_pricelist"
+                        string="Update prices from product and/or pricelist"
```

Comment:
This button string is too long in the interface (mobile render !? etc)
=> 'Update prices' is sufficient I think

Reply:
Maybe "Apply Pricelist" ?

Reply:
Or "Update prices"?